src/panels/lovelace/common/coerce-entities-to-objects.js

```diff
@@ -0,0 +1,16 @@
+/** Coerce plain entity_id strings into {entity: 'entity_id'} objects **/
+export default function coerceEntitiesToObjects(entities) {
+  if (!Array.isArray(entities))
```

Comment:
Don't do this, raise instead. Fail loudly.

Reply:
That's weird, that isn't the most recent commit. I'm not sure why you're still seeing that version. Anyway, this isn't necessarily a failure. I tried to make this as general as possible so in the most recent commit, if `entities` is not an array it still works, it just does the conversion and returns a single entity, but I am liking @c727 version better; just take the config and return the entities